CHANGES

```diff
@@ -3,24 +3,57 @@ Release 1.7 (development)
 
 .. rubric:: Enhancements
 
-* Add support for streaming installation from a HTTP(S) server for verity
-  bundles.
+* Add support for streaming installation from a HTTP(S) server for bundles in
+  ``verity`` and ``crypt`` formats.
   This avoids the need for a temporary bundle storage location and prepares for
   more efficient incremental updates.
+  See the :ref:`documentation for details <http-streaming>`.
+* Add support for bundle encryption (``crypt`` format).
+  This is useful when bundles contain confidential data and are not otherwise
+  protected during transport (for example, via HTTP, unauthenticated HTTPS or
+  USB storage).
+  See the :ref:`documentation for details <sec-encryption>`.
+* Divert log messages to stderr, which is useful for machine readable output
+  (``rauc status --output-format=json``).
+  This is only enabled when built with glib 2.68 or newer.
+  (by Ludovico de Nittis)
+* Improve the error message printed on compatible mismatch.
 
 .. rubric:: Bug fixes
```

Comment:
We've also fixed unintentional removal of existing bundles on error in 7c324a7a3dd68bd012b620bbd64c2fe822c6abde